apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -90,6 +99,113 @@ export function createDefaultSpriteMetadata(listData) {
   });
 }
 
+export function buildAnimationMetadata(files) {
+  let animationMetadataByName = {};
+  let resolvedPromisesArray = [];
+  for (const [fileKey, fileObject] of Object.entries(files)) {
+    let json = fileObject['json'];
```

Comment:
actually, on this line we should destructure `fileObject` since we're assigning 2 of its property values to local variables. the way i usually think about it:
- if i need 2+ values from an object, destructuring is more idiomatic
- always use dot notation unless bracket notation is required. example of where it would be required:

```javascript
// setting an object's key to a variable requires bracket notation
let myObj = {}
const myVar = 'hi'
myObj[myVar] = 'bonjour'

// doesn't work because key would be 'myVar' instead of 'hi'
myObj.myVar = 'bonjour'
```